.github/workflows/ci.yml

```diff
@@ -86,8 +88,8 @@ jobs:
           KUBECONFIG=${CONFIG_DIR}/kubeconfig-wascc ./target/debug/krustlet-wascc --node-name krustlet-wascc --port 3000 --bootstrap-file ${CONFIG_DIR}/bootstrap.conf --cert-file ./config/krustlet-wascc.crt --private-key-file ./config/krustlet-wascc.key &
           # Wait for things to start before approving certs and then delete so we don't overlap with the same hostname
           sleep 5 && kubectl certificate approve $(hostname)-tls
-          sleep 2 && kubectl delete csr $(hostname)-tls 
+          sleep 2 && kubectl delete csr $(hostname)-tls && sleep 2
```

Comment:
This was to solve a race condition where the delete hadn't completed (I'm not sure if `--wait` works with this either). This will be going away next week when I convert this over to our one click testing